Add Eve Room 2 Support

sieren opened the pull request:
Add support for Eve Room 2 using its characteristics
and different data stream.
The API accepts VOC in ug/m3, though the Eve app displays these
values as ppb (ug/m3 * 4.57 = ppb).

See: https://github.com/simont77/fakegato-history/issues/107

Tested with `homebridge-hue` (See https://github.com/ebaauw/homebridge-hue/pull/1020)

![IMG_040B73739D43-1](https://user-images.githubusercontent.com/5174440/132134687-e01de18b-8c90-486a-9e2a-f018888be303.jpeg)
![IMG_3492](https://user-images.githubusercontent.com/5174440/132134704-28433eb9-15a2-43fa-8791-6bd67a114003.PNG)

sieren committed:
Add Eve Room 2 Support

Add support for Eve Room 2 using its characteristics
and different data stream.
Room 2 accepts VOC in ug/m3, though the app displays these
values as ppb (ug/m3 * 4.57 = ppb).

simont77 commented:
Thanks for this PR. At the moment I’m on holiday and cannot check it, nor publish on NPM. Could you update also the readme in the meantime?

sieren commented:
@simont77 good call! Will add this.

sieren commented:
@simont77 force pushed a README update to the initial commit.

sieren commented:
Example running an Aqara TVOC Sensor via homebridge-hue:

![IMG_3499](https://user-images.githubusercontent.com/5174440/132135338-939d1222-5010-4fad-8cb3-937f56cf0abe.jpg)
![IMG_3498](https://user-images.githubusercontent.com/5174440/132135344-66a8ac98-b5df-47f6-a5c1-5b48b24a4ad3.PNG)
![IMG_3500](https://user-images.githubusercontent.com/5174440/132135345-f4836484-588f-4802-b063-0d59eaf526b1.PNG)

simont77 commented:
Are you ok with this PR or you need more testing?

sieren commented:
@simont77 it seems to look good now. I realised during long-term testing that VOC values need to be integers, not floats and the averaging by the timer would introduce floats, so I added a fix for that on tuesday specifically for VOC.
also added documentation for https://github.com/simont77/fakegato-history/issues/107#issuecomment-913822887

sieren commented:
Actually hold off, this is missing in the README.

sieren commented:
Okay, we're good now 👍
